src/web/core/src/main/java/org/geoserver/web/ServiceDescription.java

```diff
@@ -103,14 +103,14 @@ public ServiceDescription(
      * @param layer Layer name, or LayerGroup name, or {@code null} for workspace or global service
      */
     public ServiceDescription(
-            String service,
+            String serviceType,
             InternationalString title,
             InternationalString description,
             boolean available,
             boolean admin,
             String workspace,
             String layer) {
-        this.service = service.toLowerCase();
+        this.service = serviceType.toLowerCase();
```

Comment:
I think (??) this might be a bit inconsistent with rest of the code (which seems to be based on uppercase).  I don't think this will be an issue since you use `.equalsIgnoreCase`...

Reply:
Yeah I am considering changing ServiceDescription and ServiceLinkDescription to upper case just so there is less guess work.

It is good feedback I should do this; and now is the correct time to do so (when trying to make things consistent with service enablement controls).